migrations/sql-migrations/Makefile

```diff
@@ -12,4 +12,12 @@ copy-data:
 	@if [ "${REDOWNLOAD}" == "true" ]; then\
 		venv/bin/python3 dynamo_to_json.py --source-env=${SOURCE_ENV};\
 	fi
-	@SOURCE_ENV=${SOURCE_ENV} TARGET_ENV=${TARGET_ENV} npm run dynamoToSql
\ No newline at end of file
+	@SOURCE_ENV=${SOURCE_ENV} TARGET_ENV=${TARGET_ENV} npm run dynamoToSql
+
+migrate:
+	@echo "Migrating database schema for ${TARGET_ENV}"
+	@NODE_ENV=${TARGET_ENV} knex migrate:latest
+
+migrate-down:
+	@echo "Deleting database schema for ${TARGET_ENV}"
+	@NODE_ENV=${TARGET_ENV} knex migrate:down
```

Comment:
I think we'll need to change to `migrate:rollback --all`, so that it fully removes all the migrations